package.json

```diff
@@ -1,29 +1,37 @@
 {
   "name": "cbioportal-frontend",
+  "private": true,
+  "version": "0.0.0",
+  "workspaces": {
+    "packages": [".", "packages/*"]
+  },
   "description": "",
   "scripts": {
     "start": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server",
     "startSSL": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server --https",
+    "startAll": "lerna run start --parallel",
```

Comment:
runs `yarn start` for all workspaces in parallel. great feature for development.